java/connector-node/risingwave-connector-test/src/test/java/com/risingwave/connector/sink/jdbc/JDBCSinkTest.java

```diff
@@ -257,7 +282,7 @@ public void testPostgres() throws SQLException {
     @Test
     public void testMySQL() throws SQLException {
         MySQLContainer mysql =
-                new MySQLContainer<>("mysql:8")
+                new MySQLContainer<>("mysql:latest")
```

Comment:
ditto

Reply:
My local image version is `latest`, forget to change it back.